src/Microsoft.VisualStudio.Editors/OptionPages/GeneralOptionPageResources.resx

```diff
@@ -138,4 +138,7 @@
   <data name="General_UsePreviewSdk" xml:space="preserve">
     <value>Use previews of the .NET Core SDK</value>
   </data>
+  <data name="General_DoubleClickOpenProjectFile" xml:space="preserve">
+    <value>Double click on project node to edit project file.</value>
```

Comment:
This should not have a trailing ".".

Reply:
Also missing an access key.

Reply:
My WPF is weak... pointers?
![image](https://user-images.githubusercontent.com/754264/51980589-64fd8800-2488-11e9-9a12-8e1438ac86bc.png)
![image](https://user-images.githubusercontent.com/754264/51980572-5a42f300-2488-11e9-975a-db68aa814544.png)

Reply:
Use a single underscore instead of an ampersand.

Reply:
WPF is weird. Bring back Windows Forms I say!

Thanks guys :P

Reply:
`&` has a special meaning in XML. You'd have to use `&amp;`.